src/controllers/room.controller.js

```diff
@@ -0,0 +1,40 @@
+import { roomService } from '../services/room.service.js';
+import { messageService } from '../services/message.service.js';
+
+async function create(req, res) {
+  const { name } = req.body;
+  const room = await roomService.create(name);
+
+  res.status(201).send(room);
+}
+
+async function join(req, res) {
+  const { id } = req.params;
+  const room = await roomService.join(id);
+  const messages = await messageService.getMessages(id);
+
+  res.send(room, messages);
```

Comment:
The `res.send` method should only take one argument. Consider combining `room` and `messages` into a single object or array before sending the response.